code/modules/trade/datums/trade_stations_presets/1-common/hellcat.dm

```diff
@@ -78,18 +80,26 @@
 		"Explosive Weapons" = list(
 			/obj/item/gun/projectile/shotgun/pump/grenade/china,
 			/obj/item/ammo_casing/grenade/weak = custom_good_amount_range(list(6,9))
-		),
-		"Basic Gun Mods" = list(
-			/obj/item/gun_upgrade/barrel/forged,
-			/obj/item/gun_upgrade/mechanism/gravcharger,
-			/obj/item/tool_upgrade/productivity/ergonomic_grip,
-			/obj/item/tool_upgrade/refinement/laserguide
 		)
 	)
 	offer_types = list(
+		/obj/item/oddity/common/photo_crime = offer_data("crime scene photo", 500, 1),
+		/obj/item/part/armor = offer_data("armor part", 500, 8),
+		/obj/item/part/armor/artwork = offer_data("artistic armor part", 1000, 1),
+		/obj/item/part/gun/artwork = offer_data("artistic gun part", 1000, 1),
+		/obj/item/gun = offer_data_mods("modified gun (2 upgrades)", 3200, 4, OFFER_MODDED_GUN, 2),
```

Comment:
It's a really bad idea since you can just buy a cheap gun, mod it with accessories that can be bought from the same traders and just multiply the amount of money obtained

Reply:
If I was reading it right, the gun mods were removed from Hellcat, and the Techno2 station was removed, so no toolmods that can be used on guns can be bought there. You'll just have to junk for them, or bug Moebius.

Reply:
Suoh and Arau do have them still

Reply:
And there probably some gun upgrades that are now are available through loot only, such as forged barrel

Reply:
They should all be loot-only. I'm gonna make the syndi ones expensive enough to not be worth the trouble and leave the offer as-is.